pkg/storage/unified/resource/resource.proto

```diff
@@ -258,13 +258,17 @@ message ListRequest {
 }
 
 message ListResponse {
-  repeated ResourceWrapper items = 1;
+  // Error details
+  ErrorResult error = 1;
```

Comment:
@chaudyg is it too late for breaking changes?  this could go last, but then would just look weird but same behavior